Dockerfile

```diff
@@ -1,13 +1,10 @@
-# FROM mcr.microsoft.com/devcontainers/python:3.11
-
-# For the tests as it is significantly smaller
-FROM python:3.11-alpine 
+FROM mcr.microsoft.com/devcontainers/python:3.11 
```

Comment:
nope, python:3.11-alpine was correct here, this is the app container, not the devcontainer!

Reply:
done